src/ast/stmt/function-declaration.ts

```diff
@@ -10,6 +10,7 @@ import { TypeNode } from '../types';
  * Function declaration.
  */
 export interface FunctionDeclaration extends SyntaxNode {
+  readonly declareKeyword: SyntaxToken<TokenSyntaxKind> | null;
```

Comment:
@aladdin-add Look at this file. This is what makes this AST different from ESTree